samples/zigbee/light_bulb/boards/nrf21540dk_nrf52840.overlay

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
This doesn't seem like an intended change.

Reply:
Thanks, fixed